config.toml

```diff
@@ -76,6 +76,7 @@ time_format_blog = "02.01.2006"
 copyright = "The Docsy Authors"
 privacy_policy = "https://policies.google.com/privacy"
 github_repo = "https://github.com/google/docsy-example"
+github_project_repo = "https://github.com/google/docsy"
```

Comment:
This is probably better positioned beneath `github_subdir` (since `github_subdir` is related to `github_repo`)?

Also, perhaps some additional context in a comment for clarification? The documentation currently uses the term "project" to refer to the user's working clone/fork of a repo (ie, if they clone google/docsy-example, the docs refer to that as "their project"). Feel free to change but Im thinking something like:
```suggestion
# An optional link to a related project repo. For example, the sibling repository where your product code lives.
github_project_repo = "https://github.com/google/docsy"
```